### DISCUSSION TITLE
Simple Example to connect blynk local server

### DISCUSSION THREAD

**Discussion by TeguhTeknisi** [khoih-prog/ESPAsync_WiFiManager_Lite#9] [Q&A]
HI Mr Khoi Hoang, thanks for the ESPAsync_WiFiManager_Lite

I just want to have async web manager with wifi ssid, wifi password, blynk local server ip and blynk token then save it to eeprom
so how to use ths library just for thats simple web manager?

i have try to use this, but why is this need double reset detector and litle fs stuff
is there very lite version for this library?

Best Regrads,
Teguh

**Comment by khoih-prog** [comment]
Hi @TeguhTeknisi 

> save it to eeprom

For ESP32, read the code hereafter and you'll know how to change the settings to use EEPROM. Similar for ESP8266

https://github.com/khoih-prog/ESPAsync_WiFiManager_Lite/blob/1a9b2a94df6f581644c311ccc5e6fafabc584299/src/ESPAsync_WiFiManager_Lite.h#L74-L93

and

https://github.com/khoih-prog/ESPAsync_WiFiManager_Lite/blob/1a9b2a94df6f581644c311ccc5e6fafabc584299/examples/ESPAsync_WiFi/defines.h#L57-L65

---

> i have try to use this, but why is this need double reset detector and litle fs stuff
> is there very lite version for this library?

This library is written with general and more complex features to be able to handle more use-cases.

You can **easily** modify the settings and / or (open-source code) library to fit your use-case. For example, remove DRD / MRD feature, etc. by deleting / modifying certain portion of code.

I'm sorry I don't have time to spend and help you. 

Wait here for someone's help or ask on Arduino Forum if you still have issue to adapt.

Good Luck,

**Reply by TeguhTeknisi** [reply]
Thanks for the reply,

I use esp8266 ( esp12f ), but somehow its fail to compile.

For now, I will try to write my own code.

I hope someday you can make simple Async WM like IotWebConf thats support Async.

Best Regrads,
Teguh.
